packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewBuilder.java

```diff
@@ -136,6 +140,54 @@ public WebViewBuilder setDownloadListener(@Nullable DownloadListener downloadLis
     return this;
   }
 
+  /**
+   * Sets whether the {@link WebView} should use its built-in zoom mechanisms. The default value is
+   * {@code true}.
+   *
+   * @param flag {@code true} if built in zoom controls are allowed.
+   * @return This builder. This value cannot be {@code null}.
+   */
+  public WebViewBuilder setBuiltInZoomControls(boolean flag) {
```

Comment:
s/BuiltInZoomControls/EnableBuiltInZoomControls/ here and in the variable; this isn't setting any controls, just enabling or disabling them.

Reply:
Fixed in 505e912